rpc/js/.prettierrc.cjs

```diff
@@ -0,0 +1,2 @@
+// .prettierrc.cjs
```

Comment:
This filename comment was just for the README in the config repo, it doesn't have to be here 😄. Since it's a CommonJS module, we _should_ include `use strict;`, though, or else our linter will yell at us later.

```suggestion
'use strict';
```

(The "use strict;" incantation is an old silly thing from darker times in JS. Add it if the linter ever tells you to, but don't worry about it otherwise because it's on its way out)